dpsim-models/src/Base/Base_ReducedOrderSynchronGenerator.cpp

```diff
@@ -470,6 +470,10 @@ void Base::ReducedOrderSynchronGenerator<
   this->mSLog->flush();
 }
 
+template <typename VarType>
+void Base::ReducedOrderSynchronGenerator<VarType>::mnaCompApplySystemMatrixStamp(SparseMatrixRow &systemMatrix){
```

Comment:
Is your change formatted with clang-format?
```suggestion
void Base::ReducedOrderSynchronGenerator<VarType>::mnaCompApplySystemMatrixStamp(SparseMatrixRow &systemMatrix) {
```

Reply:
It was not, now will be applied